inappmessaging/src/main/java/com/rakuten/tech/mobile/inappmessaging/runtime/workmanager/workers/ConfigWorker.kt

```diff
@@ -101,7 +110,11 @@ internal class ConfigWorker(
                 Result.success()
             }
             response.code() >= HttpURLConnection.HTTP_INTERNAL_ERROR -> Result.retry() // Retry if server has error.
-            else -> Result.failure()
+            else -> {
```

Comment:
What kind of error do we except here?

Reply:
like 404 (Not Found), 403 (Forbidden) , etc.